sdcm/utils/jepsen.py

```diff
@@ -0,0 +1,79 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2021 ScyllaDB
+
+import uuid
+import logging
+from typing import Optional
+from functools import cached_property
+
+from sdcm.utils.common import list_logs_by_test_id
+from sdcm.utils.docker_utils import ContainerManager, Container, DockerException
+
+
+JEPSEN_IMAGE = "tjake/jepsen"
```

Comment:
let's pin it down...

Reply:
We can't. It's the only image in this repository: https://hub.docker.com/r/tjake/jepsen/tags?page=1&ordering=last_updated

So, even if we say `tjake/jepsen:latest` it'll not save us from problems.

Reply:
Then I think for the long run we should "branch" this image ? (and if on the process we could cache all the java dependencies, it would be also nice, but not mandatory), for now I'll merge this one.